pkg/package-server/server/server.go

```diff
@@ -194,6 +196,11 @@ func (o *PackageServerOptions) Run(ctx context.Context) error {
 		log.SetLevel(log.DebugLevel)
 	}
 
+	// Enables http2 DOS mitigations for unauthenticated clients.
+	utilfeature.DefaultMutableFeatureGate.SetFromMap(map[string]bool{
+		string(genericfeatures.UnauthenticatedHTTP2DOSMitigation): true,
+	})
+
```

Comment:
Does everyone think this is the right space to make the function call?

Reply:
Looks as reasonable as anyplace.